ocs_ci/krkn_chaos/background_cluster_operations.py

```
@@ -951,10 +950,39 @@ def _longevity_operations(self):
+                    continue
+                # Resolve to PVC instance if workload stored OCS (e.g. VdbenchWorkload.pvc)
+                if not hasattr(pvc_obj, "create_snapshot"):
+                    pvc_name = getattr(pvc_obj, "name", None)
+                    pvc_namespace = getattr(pvc_obj, "namespace", self.namespace)
+                    if pvc_name and pvc_namespace:
+                        try:
+                            resolved = pvc_helpers.get_pvc_objs(
+                                [pvc_name], namespace=pvc_namespace
+                            )
+                            if resolved:
+                                pvc_obj = resolved[0]
```

Ruff (0.15.14)

[warning] `for` loop variable `pvc_obj` overwritten by assignment target

(PLW2901)